julia/test/model_tests.jl

```diff
@@ -550,24 +550,24 @@ end
     x = [0.5] # bernoulli parameter
 
     R = 1000
-    out_size = BridgeStan.param_num(model;include_tp=false, include_gq=true)
+    out_size = BridgeStan.param_num(model; include_tp = false, include_gq = true)
 
     # to test the thread safety of our RNGs, we do two runs
     # the first we do in parallel
     gq1 = zeros(Float64, out_size, R)
-    @Threads.threads for it = 1:nt
-        rng = StanRNG(model,seeds[it]) # RNG is created per-thread
+    Threads.@threads for it = 1:nt
```

Comment:
Good catch.  I definitely would have guessed `@Threads.threads` would have errored.

Reply:
I think my formatter updated this on my behalf (or at least, it's been long enough that I'd forgotten doing it)